lib/app/modules/languages/views/language_menu.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
We'd need to move this option under settings view, it would make more sense to allow user to change his language here.

Reply:
Thank you for your review.
I've made the suggested changes, do let me know if anything else is required.